app/src/main/java/com/aboolean/movies/data/repository/PopularMoviesRepository.kt

```diff
@@ -11,13 +12,18 @@ import io.reactivex.Maybe
 import io.reactivex.Single
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.schedulers.Schedulers
+import kotlinx.coroutines.withContext
 
 interface PopularMoviesRepository : BaseRepository {
     fun getPopular(page: Int): Maybe<List<Movie>>
+    suspend fun suspendGetPopular(page : Int) : List<Movie> {
+        TODO("You must implement this method when you want to retrieve popular movies using coroutines")
```

Comment:
I think this line is not necessary because it's already implemented, what do you think?